scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -1090,75 +1095,30 @@ private void setAllValuesAsString() {
-            final VerticalPanel formContent, final FormPanel importFromCatalogformPanel) {
-        return new com.google.gwt.event.dom.client.ClickHandler() {
-            @Override
-            public void onClick(com.google.gwt.event.dom.client.ClickEvent event) {
-                // filter only valid items
-                if (bucketsListBox.getSelectedIndex() > 0 && workflowsListBox.getSelectedIndex() > 0) {
-                    String selectedWorkflowLabel = workflowsListBox.getSelectedValue();
-                    String selectedBucketName = bucketsListBox.getSelectedValue();
-                    formContent.add(new Hidden("bucketName", selectedBucketName));
-                    formContent.add(new Hidden("workflowName", selectedWorkflowLabel));
-                    displayLoadingMessage();
-                    importFromCatalogformPanel.submit();
-                }
-            }
         };
     }
 
-    private SubmitCompleteHandler fileUploadCompleteHandler() {
-        return new SubmitCompleteHandler() {
-            @Override
-            public void onSubmitComplete(SubmitCompleteEvent event) {
-                String jobEditKey = "jobEdit";
-                String res = event.getResults();
-
-                try {
-                    JSONValue js = JSONParser.parseStrict(res);
-                    JSONObject obj = js.isObject();
-
-                    if (obj.get(jobEditKey) != null && obj.get(jobEditKey).isString() != null) {
-                        String val = obj.get(jobEditKey).isString().stringValue();
-                        job = new String(org.ow2.proactive_grid_cloud_portal.common.shared.Base64Utils.fromBase64(val));
-                        // if the job has an EXECUTION_CALENDAR Generic Information defined, the startAccordingToPlanningRadioButton becomes visible, and invisible otherwise
-                        setStartAccordingPlanningRadioButtonState(job);
-                        variables = readVars(job);
-                    } else {
-                        GWT.log(JSON_ERROR);
-                        displayErrorMessage(res);
-                        //Force disable check&submit buttons to prevent confusion if a valid job was uploaded first but not submitted
-                        setEnabledStartAtPart(false);
-                        startAccordingPlanningRadioButton.setVisible(false);
-                        return;
-                    }
-
-                } catch (JSONException t) {
-                    GWT.log(JSON_ERROR);
-                    displayErrorMessage(res);
-                    //Force disable check&submit buttons to prevent confusion if a valid job was uploaded first but not submitted
-                    setEnabledStartAtPart(false);
-                    startAccordingPlanningRadioButton.setVisible(false);
-                    return;
-                }
-                redrawVariables(job);
+    private com.google.gwt.event.dom.client.ClickHandler
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)